api/v1beta1/common.go

```diff
@@ -16,7 +16,31 @@ limitations under the License.
 
 package v1beta1
 
-type CommonParameters struct {
+import (
+	"github.com/openstack-k8s-operators/lib-common/modules/common/condition"
+	corev1 "k8s.io/api/core/v1"
+)
+
+type ExtraConfigmapsMounts struct {
```

Comment:
@fmount this PR is solely about reorganizing the parameters inside the test-operator related CRs. This is a good point! I just think it can be addressed in a separate PR. If @eshulman2 finds the time to address this issue I'm would be more then happy if this got resolved but I know that she's already spent a significant amount of time on making the jobs happy.

I'm still going to do a proper review after the recent changes but unless we find something significant then from my point of view this is good to be merged.

Reply:
as lukas mentioned this is a reorg, @fmount please raise a jira ticket for this comment to make sure it won't get lost, I'm going to resolve this thread for now

Reply:
Done: https://issues.redhat.com/browse/OSPRH-10366 @lpiwowar I assigned it to you but feel free to revisit that jira and coordinate the requiered work.

Reply:
Thanks @fmount !